lib/Capabilities.php

```diff
@@ -145,6 +146,10 @@ public function getCapabilities(): array {
 			$capabilities['features'][] = 'reactions';
 		}
 
+		if ($this->talkConfig->isRecordingEnabled()) {
+			$capabilities['features'][] = 'recording';
```

Comment:
This is supposed to be a config, not a feature. The recording-v1 is the feature. Just search for `breakout` in this file after rebasing on latest master for how it should be